python-sdk/nuscenes/eval/lidarseg/evaluate.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+from tqdm import tqdm
+import numpy as np
+
+from nuscenes import NuScenes
+
+from utils import LidarsegChallengeAdaptor
+
+
+class LidarSegEval:
+    def __init__(self,
+                 nusc: NuScenes,
+                 results_folder: str,
+                 ignore_idx: int = 0):
```

Comment:
Better to work with the name to make it independent of the mapping that is used.

Reply:
Removed need for user to specify `ignore_idx`; everything is defined once in the adaptor class now